### DISCUSSION TITLE
What documentation should be updated when contributing to the project?

### DISCUSSION THREAD

**Discussion by scottwestover** [GDQuest/learn-gdscript#523] [Q&A] [answered]
When opening a pull request to this repository, there is a checklist item for: `You updated the docs or changelog`. However, when I reviewed the existing changelog it looks like it is only updated when there is a version release.

Should we just add our notes at the top of the file in an `unreleased` section, or is there a different approach I should follow?

**Comment by NathanLovato** [comment, marked as answer]
We should remove that from the issue template actually. These days we use a little program to generate the changelog.

**Reply by scottwestover** [reply]
Very nice! Thanks for your quick response.
